content/javascript/concepts/objects/terms/create/create.md

```diff
@@ -0,0 +1,47 @@
+Title: 'Object.create() in JavaScript'
+Description: 'static method creates a new object, using an existing object as the prototype of the newly created object.It basically allows you to create a new object that inherits properties and methods from a specified prototype object. It's not about making a copy of an existing object, but rather establishing a prototype chain.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+  - 'javascript'
+  - 'objects'
+  - 'prototype'
+   Tags:
+  - 'javascript'
+  -  'objects'
+  - 'inheritance'
+  - 'prototype'
+  - 'Documentation'
+  CatalogContent:
+  - 'javascript-objects'
+  - 'paths/computer-science'
+
+  ## Syntax
+ ```js
+Object.create(proto)
+Object.create(proto, propertiesObject)
+ ```
+  The **Object.create()** method creates a new object with the specified prototype object and properties. If no prototype is provided, the newly created object will have null as its prototype. This means that the newly created object will not inherit any properties or methods from any other object.
+  The `propertiesObject` parameter is optional and allows you to add additional properties to the newly created object.You can specify which properties you want to add to the new object. For example, you might want to add a property called name, age, or anything else.
+
+
+## Example
+Lets take a look at an example that explains this concept better
+```js
+// Creating an object to serve as the prototype
+var prototypeObject = {
+  greet: function() {
+    return 'Hello!';
+  }
+};
+
+// Creating a new object with prototypeObject as its prototype
+var newObj = Object.create(prototypeObject);
+
+// newObj inherits the greet function from prototypeObject
+console.log(newObj.greet()); // Output: "Hello!"
+```
```

Comment:
Please leave a line after this, and also use the following after leaving the line:

It produces the following output:
```shell
Hello!
```